src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionPanel.cs

```diff
@@ -2382,7 +2382,7 @@ protected override CreateParams CreateParams
                     {
                         CreateParams cp = base.CreateParams;
                         cp.ExStyle |= (int)User32.WS_EX.TOOLWINDOW;
-                        cp.Style |= NativeMethods.WS_POPUP | NativeMethods.WS_BORDER;
+                        cp.Style |= unchecked((int)User32.WS.POPUP) | (int)User32.WS.BORDER;
```

Comment:
@gpetrou I think it is slightly less verbose and easier on an eye to cast once:
```suggestion
                        cp.Style |= unchecked((int)(User32.WS.POPUP | User32.WS.BORDER);
```
The same applies to other cases

@JeremyKuhne a risk of an error in this way is probably higher than the old way, as now the caller must remember to call `unchecked`.

Reply:
You don't need to use unchecked, and in fact, shouldn't. All runtime casts are unchecked, only constants are checked (i.e. compile time only).

Reply:
It doesn't work without `unchecked` though
![image](https://user-images.githubusercontent.com/4403806/67727419-75ee2080-fa3d-11e9-96ee-ef567a3422d8.png)

So I guess my worry was unfounded, the compiler will err and warn about the lack of `unchecked`.

Reply:
I have updated the code based on your suggestions.